src/bundle/DependencyInjection/Configuration/Parser/UserProfile.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+/**
+ * @copyright Copyright (C) Ibexa AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace Ibexa\Bundle\AdminUi\DependencyInjection\Configuration\Parser;
+
+use Ibexa\Bundle\Core\DependencyInjection\Configuration\AbstractParser;
+use Ibexa\Bundle\Core\DependencyInjection\Configuration\SiteAccessAware\ContextualizerInterface;
+use Symfony\Component\Config\Definition\Builder\NodeBuilder;
+
+final class UserProfile extends AbstractParser
+{
+    public function addSemanticConfig(NodeBuilder $nodeBuilder): void
+    {
+        $nodeBuilder
+            ->arrayNode('user_profile')
+                ->addDefaultsIfNotSet()
+                ->children()
+                    ->booleanNode('enabled')
+                        ->defaultFalse()
+                    ->end()
+                    ->arrayNode('content_types')
```

Comment:
what is the point of this configuration? Cant we just not relay on `user_content_type_identifier` parameter? I think introducing another one would just add confusion, even more when this is site access aware, and we decided that this dosent have much sense in terms of defining content types.

Unless, there is something else behind this config, then please, elaborate.
@adamwojs

Reply:
We want to enable user profile only for marketer persona, what is only small part of the user base.